localization/english/event_localization/wc_scourge_invasion_l_english.yml

```diff
@@ -4,4 +4,16 @@
  WCCSC_5_OPT_A_death:0 "All are equal in death."
  WCCSC_5_OPT_A_legion:0 "This is your last chance, [nerzhul.GetFirstName]. Serve, or be destroyed."
  WCCSC_5_OPT_A_evil:0 "Death has come into play at last..."
- WCCSC_5_OPT_A_neutral:0 "[ROOT.Char.GetFaith.HighGodName|U] save us all!"
\ No newline at end of file
+ WCCSC_5_OPT_A_neutral:0 "[ROOT.Char.GetFaith.HighGodName|U] save us all!"
+ 
+ WCCSC_15_TITLE:0 "Jailors of the Damned"
+ WCCSC_15_DESC:0 ""Did you think of yourself shrewd enough to get rid of us, mortal?" [scourge_jailor.GetFirstName] asked. A hint of snark in [scourge_jailor.GetHerHis] voice as [scourge_jailor.GetHerHis] fiery eyes laid on you. Angered, powerless is how you felt.\n\nYou'd somehow manage to get rid of one of [scourge_jailor.GetHerHis] brothers, perhaps in a foolish bid to escape the gilded cage in which the Legion trapped you. Perhaps you should've stuck with doing what you were told to do, but there you were: powerless. As the demonic being tore the Frozen Throne apart, you pondered what could have been if you had succeeded."
+ RULKAN_OPT:0 "Rulkan, my love, I'm coming..."
```

Comment:
I think it's so funny "Rulkan, my love, I'm coming..."
I think this is a bit out of character, at this moment Ner'zhul is a power hungry mongrel, i doubt he would think back to Rulkan, but as i said i don't hate it. i think it could be this way although as i said it could be better, but i dunno with what.

Perhaps something to do with draenor being destroyed and thinking about what ever the halls of his ancestors still exist. Like "will I join my ancestors in the afterlife?... or does only nothingness awaits me"

Reply:
> I think it's so funny "Rulkan, my love, I'm coming..."
> I think this is a bit out of character, at this moment Ner'zhul is a power hungry mongrel, i doubt he would think back to Rulkan, but as i said i don't hate it. i think it could be this way although as i said it could be better, but i dunno with what.

He did think back.
https://wowpedia.fandom.com/wiki/Ner'zhul#Shadowlands

Reply:
Haven't finished shadowllands and don't wanna spoil myself but i take your word for it ;)